src/Resources/ContactProperties.php

```diff
@@ -107,7 +107,30 @@ public function delete($name)
      */
     public function getGroups($includeProperties = false)
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/groups';
+        $endpoint = "https://api.hubapi.com/contacts/v2/groups";
```

Comment:
Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)